packages/medusa/src/loaders/index.ts

```diff
@@ -176,10 +176,10 @@ export default async ({
   const dAct = Logger.success(defaultsActivity, "Defaults initialized") || {}
   track("DEFAULTS_INIT_COMPLETED", { duration: dAct.duration })
 
-  const searchActivity = Logger.activity("Initializing search engine indexing")
+  const searchActivity = Logger.activity("Emitting search engine indexing event")
   track("SEARCH_ENGINE_INDEXING_STARTED")
   await searchIndexLoader({ container })
-  const searchAct = Logger.success(searchActivity, "Indexing completed") || {}
+  const searchAct = Logger.success(searchActivity, "Indexing event emitted") || {}
   track("SEARCH_ENGINE_INDEXING_COMPLETED", { duration: searchAct.duration })
```

Comment:
can we keep the `"Initializing search engine indexing"` logged before initializing indexing and just change the latter string to something like `"Indexing initialized"`?

Reply:
But it does not initialise the indexing anymore, it just emit an event for the subscriber to manage the indexing. If you dont have the notion of event emitted, the users that does not start there redis will not understand why the indexing does not work? wdyt?

Reply:
Yeah, I didn't try to indicate we did the indexing right away, just that we initialized the indexing. We can keep the even emitting terminology if we deem it more descriptive for the user 😄

Reply:
is it better now?

Reply:
Looks good 😄